### DISCUSSION TITLE
新年好呀，哪一个版本适合linux？

### DISCUSSION THREAD

**Discussion by earthmoon** [xiaoyifang/goldendict-ng#1347] [Q&A]
新年好呀，
[这个链接](https://github.com/xiaoyifang/goldendict-ng/releases/tag/v23.12.07-HeavySnow.b0086d5c)里面的所有版本，貌似分别是给windows和macos用的，哪一个适合linux呢？
![Screenshot from 2024-01-01 15-14-29](https://github.com/xiaoyifang/goldendict-ng/assets/45897471/7aee3b09-f7d6-4b0e-b693-6d8b77eefe35)

谢谢。

**Comment by xiaoyifang** [comment]
![image](https://github.com/xiaoyifang/goldendict-ng/assets/105986/0e90f929-f42b-4104-8981-275233c57116)

**Reply by earthmoon** [reply]
非常谢谢。新版本（Flatpak）比老版本好看多了，但是为什么速度慢了很多呢？

**Reply by xiaoyifang** [reply]
取决于你用了什么词典

**Comment by earthmoon** [comment]
词典没变，一模一样。不过现在这个版本在开机后1小时后显著变快，但还是没有老版快。
________________________________
